src/screens/WhotoFollow/index.js

```diff
@@ -245,11 +249,12 @@ const WhotoFollow = () => {
             backgroundColor: COLORS.signed_primary,
             style: top > 55 ? {paddingTop: 50} : undefined
           });
+          setIsUserRegistered(true);
           setTimeout(() => {
             create();
             setImage(null, usersDispatch);
             setInitialValue({id: res.token});
-          }, 2000);
+          }, 500);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Reducing the timeout duration from 2000ms to 500ms can improve the responsiveness of your application. However, ensure that this change doesn't introduce any race conditions or negatively impact the user experience due to the faster execution time. Also, when setting the `isUserRegistered` state, consider persisting this information in a more permanent location to maintain consistency even if the component unmounts and remounts.

```diff
  setIsUserRegistered(true);
+ // Persist the registration status in a more permanent location
+ // This is just a placeholder. Replace with actual implementation.
+ localStorage.setItem('isUserRegistered', 'true');
  setTimeout(() => {
    create();
    setImage(null, usersDispatch);
    setInitialValue({id: res.token});
  }, 500);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->